config/config_test.go

```diff
@@ -58,3 +70,61 @@ func TestDurationMarshalJSON(t *testing.T) {
 	assert.Equal(t, `"2s"`, string(data))
 	assert.NoError(t, err)
 }
+
+func TestFromConfig(t *testing.T) {
+	timeout := 20 * time.Second
+	backend := fiber.NewBackend("proxy_name", "localhost:1234")
+
+	httpDispatcher, _ := fiberhttp.NewDispatcher(&http.Client{Timeout: timeout})
+	httpCaller, _ := fiber.NewCaller("proxy_name", httpDispatcher)
+	httpProxy := fiber.NewProxy(backend, httpCaller)
+	testutils.RunTestUPIServer(testutils.GrpcTestServer{
+		Port: port,
+	})
+
+	grpcDispatcher, _ := fibergrpc.NewDispatcher(
+		fibergrpc.DispatcherConfig{
+			Service:           "testproto.UniversalPredictionService",
+			Method:            "PredictValues",
+			ResponseProtoName: "PredictValuesResponse",
```

Comment:
Does `ResponseProtoName` expect only proto name? or a fully qualified name? How does it handle when 2 protos have same name but come in different package?

Reply:
currently it using `strings.contain` so both will work, but when there are multiple same name proto, then its indeterministic as you had pointed out. 

I can change it to check for equality, but user would have to provide the full package name.

Reply:
removed this altogether, thanks for the suggestion.

the package is crossed check as part of config.service and outtype inferred from the same file